### ISSUE TITLE
Using Zongji to monitor mysql live event

### ISSUE THREAD

**Issue by shtse8** [joeybaker/sequelize-stream#14]
I had an idea to make this project more interesting.
We can use [ZongJi](https://github.com/nevill/zongji) to monitor mysql binlog event to reload the sequelize model so that we can emit the changes.

**Comment by joeybaker** [comment]
That's an interesting idea, but I'm not sure it would pair well with sequelize since it's an adaptor for PostgreSQL, SQLite and MSSQL in addition to MySQL. Thoughts?

**Comment by shtse8** [comment]
Ya, you are right.
this is the only problem ZongJi is just for MySQL server binlog tracking.
I know it's hard to make it perfect for all types of databases.
But I think we should and still do better as we can.
At the moment, in my project, I am using Sequelize for Mysql Models, ZongJi for Mysql created/updated/destroyed events, and this project `Sequelize-Stream` for model instances event.
my project is working great, and all components are working great. But sadly, I need to implement a lots to make these three components to work great with each other. So I think it's better to merge them up and become a amazing and interesting project in open source on github for more people who want to do the same thing. Any thoughts?

**Comment by joeybaker** [comment]
Sounds like you might want to write that module. Maybe `sequelize-stream-mysql`?

I'm a little unsure why you need ZongJi though? Are you running multiple app instances and using that to keep things in sync?

**Comment by shtse8** [comment]
Yes, for large scale project and better structural projects. We should always separate services and setup on different servers and locations for better performance.

**Comment by joeybaker** [comment]
Gotcha. I'm running Postgres, and took a slightly different approach:

Every app server uses `sequelize-stream` to broadcast all changes to RabbitMQ with the event type, instance JSON, and changes. RabbitMQ then re-broadcasts this to all other app servers which consume that change however they need to. It means we have a pretty hard dependency on RabbitMQ, but it works well.
